apps/backend/api/graphql/mutations/comment.js

```diff
@@ -11,9 +11,12 @@ export async function canDeleteComment (userId, comment) {
 
   const commentWithGroups = await comment.load('post.groups')
 
-  return commentWithGroups.relations.post.relations.groups.map(g =>
+  const hasResponsibilityPromises = commentWithGroups.relations.post.relations.groups.models.map(g =>
```

Comment:
still unclear on whether this was broken before, just making sure you are testing the functionality too